app/src/main/java/com/gpayinterns/chat/ServerHelper.java

```diff
@@ -0,0 +1,20 @@
+package com.gpayinterns.chat;
+
+public class ServerHelper
+{
```

Comment:
Please add a private constructor to this class.
This is not supposed to be instantiated.